File: New folder/Q5.c
// 5. Write a function that checks if a person is eligible to vote. The person is eligible if they are 18 years old or older.



#include <stdio.h>
int main(){
    int age;
    printf("Enter your age:");
    scanf("%d",&age);
    if(age>=18){
        printf("Eligible for Vote");
    }else{
        printf("not Eligible for vote");
    }
}

File: Java Script method/Q5.js
// 5. Write a function that checks if a person is eligible to vote. The person is eligible if they are 18 years old or older.

function age(yourage){
    if(yourage>=18){
        console.log("Person is eligible for vote")
    }else{
        console.log("Person is not eligibal for vote")
    }
}
console.log(age(18))